internal/host/host.go

```diff
@@ -312,6 +313,24 @@ func (m *Manager) UpdateInventory(ctx context.Context, h *models.Host, inventory
 	}).Error
 }
 
+func (m *Manager) checkValidationChanged(ctx context.Context, h *models.Host, newValidationRes validationsStatus) (validationsStatus, bool, error) {
+	var currentValidationRes validationsStatus
+	if h.ValidationsInfo != "" {
```

Comment:
Won't it be simpler to check string equality on json encoded strings?

Reply:
This will require Marshalling the new validation results, which can fail on its own, so I don't think it is easier.